frameworks/elastic/universe/marathon.json.mustache

```diff
@@ -55,6 +56,7 @@
     {{/service.virtual_networks}}
 
     "DEPLOY_STRATEGY": "{{service.deploy_strategy}}",
+    "TASKCFG_ALL_JAVA_HOME": "jre1.8.0_131/",
```

Comment:
Same as above. Should do this dynamically within the `cmd` so that it's less work to bump the JRE (happens every month or two), and/or to even allow in-the-field upgrades of the JRE.